scimath/Mathematics/ClassicalStatistics.tcc

```diff
@@ -908,25 +905,19 @@ void ClassicalStatistics<CASA_STATP>::_accumulate(
 
 CASA_STATD
 uInt ClassicalStatistics<CASA_STATP>::_nThreadsMax() const {
+    uInt nthr = OMP::nMaxThreads();
 #ifdef _OPENMP
```

Comment:
Is this necessary any longer?

Reply:
the ifdef probably isn't required. I'm working on another stats related issue atm, so if this isn't required, I'll remove it on that branch.
Thanks for the review.